mmf/datasets/builders/coco/dataset.py

```diff
@@ -38,9 +38,12 @@ def load_item(self, idx):
 
         current_sample.image_id = object_to_byte_tensor(sample_info["image_id"])
 
-        if self._use_features is True:
+        if self._use_features:
             features = self.features_db[idx]
             current_sample.update(features)
+        else:
+            image_path = str(sample_info["image_name"]) + ".jpg"
+            current_sample.image = self.image_db.from_path(image_path)["images"][0]
```

Comment:
The transforms should be assigned to image_db so they are applied automatically.

Reply:
I believe this needs to be fixed before this can land.

Reply:
Thanks, amended!